Source/SwiftLintCore/Extensions/SwiftSyntax+SwiftLint.swift

```diff
@@ -184,6 +184,25 @@ public extension EnumDeclSyntax {
             return rawValueTypes.contains(identifier)
         }
     }
+
+    /// True if this enum is a `CodingKey`
+    var isCodingKey: Bool {
+        guard let inheritedTypeCollection = inheritanceClause?.inheritedTypes else {
+            return false
+        }
+        
+        guard self.name.text == "CodingKeys" else {
```

Comment:
```suggestion
        guard name.text == "CodingKeys" else {
```

Can also be added to the previous `guard`.